setup.py

```diff
@@ -28,8 +28,8 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/vikinganalytics/mvg",
-    package_dir={"": "mvg"},
-    packages=setuptools.find_packages(where="mvg"),
+    package_dir={"": "."},
```

Comment:
Is this row even required now?

Reply:
Not really. It just resembles the common practice now. In many packages there is a `src` folder that contains the modules. In that setup they use 
```
    package_dir={"": "src"},
    packages=setuptools.find_packages(where="src")
```

For us this the root because `mvg` package lives directly in the root. We can remove it but I wold rather keep it to match the common practice. Easier to grasp for someone familiar with that.